torch/export/experimental/__init__.py

```diff
@@ -67,3 +69,58 @@ def _export_forward_backward(
     _remove_detach_pass(gm, new_graph_signature)
 
     return ep._update(gm, new_graph_signature)
+
+
+def _sticky_export(model):
```

Comment:
I intentinally didn't make it a decorator because that would mean we need to change user code. Can extend it for decorator tho, if ppl find it more useful.